### PR TITLE
V7 2357 import classifications as tags

### PR DESCRIPTION
**Pull request by Fl4m3Ph03n1x**
What?
-----
This PR aims to convert labelbox classifications into darwin.py annotation tags. 

Basically speaking, given an object with the following format:

```
{
            "Label":{
                "objects":[
                    {
                        "title": "SomeTitle",
                        "bbox":{"top":3558, "left":145, "height":623, "width":449}
                    },
                    {...}
                ],
                "classifications": [
                    {
                        "value": "a_question",
                        "answer": {"value": "an_answer"}
                    },
                    ....
                ]
            },
            "External ID": "demo-image-7.jpg"
        },
        {...}
```

We must be able to convert the `classifications` array into Annotation Tags.

How can I test this?
-----

I have added tests so you can simply run `pytest` and everything should work.
Alternatively you can also do an import against a dataset you own:
```
darwin dataset import DATASET_NAME labelbox test.json
```

Where `test.json` is:

```
[
    {
        "Label": {
            "objects": [
                {
                    "title": "UglyCar",
                    "polygon": [
                        {
                            "x": 3665.814,
                            "y": 351.628
                        },
                        {
                            "x": 3762.93,
                            "y": 810.419
                        },
                        {
                            "x": 3665.814,
                            "y": 351.628
                        }
                    ]
                }
            ],
            "classifications": [
                {
                    "value": "is_ugly",
                    "answer": {
                        "value": "yes"
                    }
                }
            ]
        },
        "External ID": "IMAGE_NAME"
    }
]
```

### CODE REVIEW

File: darwin/importer/formats/labelbox.py [hunk 1]
```diff
@@ -43,6 +53,12 @@ def parse_file(path: Path, validate: Callable[[Any], None]) -> Optional[List[Ann
     Currently we support the following annotations:
     - bounding-box `Image`: https://docs.labelbox.com/docs/bounding-box-json
     - polygon `Image`: https://docs.labelbox.com/docs/polygon-json
+    - point `Image`: https://docs.labelbox.com/docs/point-json
+    - polyline `Image`: https://docs.labelbox.com/docs/polyline-json
+
+    We also support conversion from question/answer to Annotation Tags for the following:
+    - Radio Buttons
+    - Checklists
```
Comment (andreaazzini) [inline darwin/importer/formats/labelbox.py:61]: should we add free text here?

File: darwin/importer/formats/labelbox.py [hunk 2]
```diff
@@ -74,15 +90,22 @@ def parse_file(path: Path, validate: Callable[[Any], None]) -> Optional[List[Ann
...
 
 
 def _convert(file_data: Dict[str, Any], path) -> AnnotationFile:
     filename: str = str(file_data.get("External ID"))
     label: Dict[str, Any] = cast(Dict[str, Any], file_data.get("Label"))
     label_objects: List[Dict[str, Any]] = cast(List[Dict[str, Any]], label.get("objects"))
+    label_classifications: List[Dict[str, Any]] = cast(List[Dict[str, Any]], label.get("classifications"))
+
+    classification_annotations: List[Annotation] = []
+    if label_classifications != []:
```
Comment (andreaazzini) [inline darwin/importer/formats/labelbox.py:103]:
```suggestion
    if len(label_classifications) > 0:
```
is more pythonic

**Review by andreaazzini — CHANGES_REQUESTED** [review]
LGTM, minor changes, but can be merged once resolved

**Review by FrancescoSaverioZuppichini — APPROVED** [review]
amazing work, LGTM!

File: darwin/importer/formats/labelbox.py (hunk 1 above)
Comment (Fl4m3Ph03n1x) [reply darwin/importer/formats/labelbox.py:61]: done!

File: darwin/importer/formats/labelbox.py (hunk 2 above)
Comment (Fl4m3Ph03n1x) [reply darwin/importer/formats/labelbox.py]: done

**Review by andreaazzini — APPROVED** [review]

**Pull request merged**